### DISCUSSION TITLE
Big Bird Day 2023 Stats

### DISCUSSION THREAD

**Discussion by timsterc** [mcguirepr89/BirdNET-Pi#910] [Show and tell]
Hi BirdNET-Pi Gang,

Just wanted to say thanks again for another very successful [Global Big Day!](https://www.birdweather.com/top/bigday)
Obviously couldn't do it without all the continued support and enthusiasm in the BirdNET-Pi community!

... and suffice it to say, I'm *very* jealous of the Karney Ranch, VA station!

Thank You All!
-Tim
Founder - BirdWeather

**Comment by lloydbayley** [comment]
Nice one Tim! Thanks for letting us know!
I didn't make the list (Station 200) but you can't have everything, can you? :)

It makes for interesting reading and you're right, Karney Ranch seems to be mounted on a bird-meeting pedestal! :)
